README.rst

```diff
@@ -317,6 +317,20 @@ Load kernel modules and add them to `/etc/modules`:
             - tp_smapi
             - 8021q
 
+Configure kernel modules with additional options to `/etc/modprobe.d`
+following example will add `/etc/modprobe.d/nf_conntrack.conf` file
+with line `options nf_conntrack hashsize=262144`:
+
+.. code-block:: yaml
+
+    linux:
+      system:
+        kernel:
+          kmod_config:
```

Comment:
I am not sure, that i have understood you correctly, however if i did:
/etc/modprobe.d directory has a lot ofoptions, including multi-line custom scripts. The structure i've chosen, allows to add lines one by one, exactly like needed, because it supports multi line scripts in ordered fashion, and i would really like to leave it as is. 

I also think that  changing kernel modules/parameters is specific and advanced feature, that should have as much flexibility as there can be.